scripts/ci/test_pytorch_blade.sh

```diff
@@ -29,8 +29,10 @@ export TF_REMOTE_CACHE=${TF_REMOTE_CACHE}
 # note(yancey.yx): using virtualenv to avoid permission issue on workflow actions CI,
 python -m virtualenv venv && source venv/bin/activate
 
+export TORCH_BLADE_CI_BUILD_TORCH_VERSION=${TORCH_BLADE_CI_BUILD_TORCH_VERSION:-1.7.1+cu110}
 (cd pytorch_blade \
-  && python -m pip install -q -r requirements-dev-1.7.1+cu110.txt -f https://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/pytorch/wheels/repo.html  \
+  && python -m pip install -q -r requirements-dev-${TORCH_BLADE_CI_BUILD_TORCH_VERSION}.txt \
+       -f https://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/pytorch/wheels/repo.html \
```

Comment:
How about depending on official pypi index?

Reply:
The index-url might be chosen because it's stable? Anyway, this PR only do refactoring. It should be updated in another PR if the official index is preferred.